web/src/app/services/UniversalKey/DefaultActionEditDialog.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react'
+import { gql, useMutation, useQuery } from 'urql'
+import FormDialog from '../../dialogs/FormDialog'
+import { ActionInput, IntegrationKey } from '../../../schema'
+import { getNotice } from './utils'
+import UniversalKeyActionsForm from './UniversalKeyActionsForm'
+import { useErrorConsumer } from '../../util/ErrorConsumer'
+
+interface DefaultActionEditDialogProps {
+  keyID: string
+  onClose: () => void
+}
+
+const query = gql`
+  query UniversalKeyPage($keyID: ID!) {
+    integrationKey(id: $keyID) {
+      id
+      config {
+        defaultActions {
+          dest {
+            type
+            args
+          }
+          params
+        }
+      }
+    }
+  }
+`
+
+const mutation = gql`
+  mutation ($input: UpdateKeyConfigInput!) {
+    updateKeyConfig(input: $input)
+  }
+`
+
+export default function DefaultActionEditDialog(
+  props: DefaultActionEditDialogProps,
+): JSX.Element {
+  const [q] = useQuery<{
+    integrationKey: IntegrationKey
+  }>({
+    query,
+    variables: {
+      keyID: props.keyID,
+    },
+  })
+
+  const [value, setValue] = useState<ActionInput[]>(
+    q.data?.integrationKey.config.defaultActions ?? [],
+  )
+  const [m, commit] = useMutation(mutation)
```

Comment:
nit, for clarity:

name `m` something such as `status`, or destructure them inplace, e.g. `const [{ data, fetching, error }, commit]`